### DISCUSSION TITLE
Is running seaweedfs with SSE-C possible?

### DISCUSSION THREAD

**Discussion by themrdanielvp** [seaweedfs/seaweedfs#4931] [Q&A] [answered]
Hello.

We are currently in the market for a new distributed file system. Can any of you tell me if running seaweedfs with SSE-C is possible? And if so maybe point me towards some documentation or other info?

I have tried to find anything about it myself but have not been able to.

**Comment by kmlebedev** [comment, marked as answer]
> Hello.
> 
> We are currently in the market for a new distributed file system. Can any of you tell me if running seaweedfs with SSE-C is possible? And if so maybe point me towards some documentation or other info?
> 
> I have tried to find anything about it myself but have not been able to.

Not at the moment, but for now you can run it via CSE-KMS.
See https://github.com/seaweedfs/seaweedfs/wiki/rclone-with-SeaweedFS#client-side-encryption

Theoretically, you need to integrate part of the code from local KMS and the rest will take off on its own.

**Comment by chrislusf** [comment]
SSE-C is supported now.
